ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -345,8 +348,7 @@
                             {:long-name long})
                           (when-let [headsign (:gtfs/trip-headsign route)]
                             {:headsign headsign})
-                          (when-let [route-hash-id (:gtfs/route-hash-id route)]
-                            {:route-hash-id route-hash-id}))
+                            {:route-hash-id route-hash-id})
```

Comment:
Minor code layout issue for `{:route-hash-id route-hash-id}`, could be more left & on top